### PR TITLE
Added Lumen support

### PR DESCRIPTION
**Pull request by muhammadnm**
Added Lumen support and tested with Country (mainly I need this only). Would be happy to hear your comments on it!

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +30,17 @@ composer require propaganistas/laravel-intl
...
-
+#### Laravel
  ```php
 'providers' => [
     ...
     Propaganistas\LaravelIntl\IntlServiceProvider::class,
 ],
 ```
+#### Lumen
+ ```php
```
Comment (Propaganistas) [inline README.md]: Please add the following line right above the code snippet:

```
In `bootstrap/app.php`, register the Service Provider
```

File: README.md [hunk 2]
```diff
@@ -30,13 +30,17 @@ composer require propaganistas/laravel-intl
 ```
 
 In your app config, add the Service Provider to the `$providers` array
```
Comment (Propaganistas) [inline README.md]: Please move this line below the "Laravel" header

File: README.md [hunk 3]
```diff
@@ -45,7 +49,7 @@ In your app config, add the Service Provider to the `$providers` array
 #### Country
 
 Output localized country names.
-
+#### Laravel
```
Comment (Propaganistas) [inline README.md]: No distinction in use required. Please remove.

File: README.md [hunk 4]
```diff
@@ -65,7 +69,16 @@ country('US'); // United States
 country()->currency('US'); // USD
 country()->all(); // ['US' => 'United States', 'BE' => 'Belgium', ...]
 ```
+#### Lumen
```
Comment (Propaganistas) [inline README.md]: No distinction in use required. Please remove this block.

File: src/IntlServiceProvider.php [hunk 5]
```diff
@@ -116,9 +116,15 @@ protected function registerDateHandler()
...
 
-        $this->app->booted(function ($app) {
-            \Jenssegers\Date\Date::setFallbackLocale($app['config']['app.fallback_locale']);
-        });
+        if (method_exists($this->app, 'booted')) {
+            $this->app->booted(function ($app) {
+                \Jenssegers\Date\Date::setFallbackLocale($app['config']['app.fallback_locale']);
+            });
+        } else {
+            call_user_func(function ($app) {
```
Comment (Propaganistas) [inline src/IntlServiceProvider.php]: I don't see why wrapping this call in a `call_user_func()` is necessary?

Comment (muhammadnm) [reply src/IntlServiceProvider.php]: I also was not sure, but thought it would be useful if there's any use with Date, so I put that

Comment (Propaganistas) [reply src/IntlServiceProvider.php]: Nah, remove it and call the Date library directly.

File: README.md (hunk 4 above)
Comment (muhammadnm) [reply README.md]: For Lumen there's some trick to set locale 
Country::setLocale('hi-IN');

How about if I remove distinction and add note
> Note: To set Locale in Lumen, you need do like ```Country::setLocale('hi-IN');```

File: src/IntlServiceProvider.php (hunk 5 above)
Comment (muhammadnm) [reply src/IntlServiceProvider.php]: Ok removed and just put this
```\Jenssegers\Date\Date::setFallbackLocale($app['config']['app.fallback_locale']);```

File: README.md (hunk 4 above)
Comment (Propaganistas) [reply README.md]: The application's locale is automatically set in the Service Provider. There's another section at the end of the README explaining how to change the locale nonetheless. So there's really no need to mention `setLocale()` here as it's mentioned further below.

File: src/IntlServiceProvider.php [hunk 6]
```diff
@@ -116,15 +116,7 @@ protected function registerDateHandler()
...
-        if (method_exists($this->app, 'booted')) {
-            $this->app->booted(function ($app) {
-                \Jenssegers\Date\Date::setFallbackLocale($app['config']['app.fallback_locale']);
-            });
-        } else {
-            call_user_func(function ($app) {
-                \Jenssegers\Date\Date::setFallbackLocale($app['config']['app.fallback_locale']);
-            }, $this->app);
-        }
+        \Jenssegers\Date\Date::setFallbackLocale($app['config']['app.fallback_locale']);
```
Comment (Propaganistas) [inline src/IntlServiceProvider.php]: Should be
```php
\Jenssegers\Date\Date::setFallbackLocale($this->app['config']['app.fallback_locale']);
```

**Comment by muhammadnm** [comment]
Alright, fixed that, sorry it was mistaken, due to hurry!

**Comment by Propaganistas** [comment]
Add support for Lumen

**Comment by Propaganistas** [comment]
Woops sorry, wrong button.

**Pull request merged**